CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to `yii2-imperavi-widget` will be documented in this file.
 ## 2.0.6
 
 ### Changes
+- Fix the issue #130 add a custom clips throw settings (bscheshirwork) 
```

Comment:
@bscheshirwork thank you for your contribution.
The only comment I have for this PR is this line.
Can you please move it to a new version: `2.0.7` along with separate `Changed` section?
Thanks!